data/crac/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/CracImplTest.java

```diff
@@ -595,7 +617,90 @@ public void testRemoveRangeAction() {
     }
 
     @Test
-    public void testFilterRangeActionUsageRules() {
+    public void testRemoveHvdcRangeAction() {
+
+        crac.newContingency().withId("co1").withNetworkElement("neCo").add();
+        crac.newContingency().withId("co2").withNetworkElement("neCo").add();
+
+        HvdcRangeAction ra1 = crac.newHvdcRangeAction()
+                .withId("ra1")
+                .withNetworkElement("ne1")
+                .newOnStateUsageRule().withUsageMethod(AVAILABLE).withContingency("co1").withInstant(CURATIVE).add()
+                .newHvdcRange()
+                .withMin(-5)
+                .withMax(10)
+                .add()
```

Comment:
it's just a detail, but what we usually do in that case - for lisibility - is to either : 
* put all the newHvdcRange() addition on the same line
* or, re-indent those three lines.